src/ripple/test/jtx/impl/Env.cpp

```diff
@@ -50,17 +50,41 @@ namespace test {
 
 namespace jtx {
 
+Env::AppBundle::AppBundle(Application* app_)
+    : app (app_)
+{
+}
```

Comment:
This doesn't look like it's used anywhere. Is it "part of a framework to create multiple Application objects in the same unit test and connect them together" that you mention in the PR description?